cosmos/dbt/graph.py

```diff
@@ -194,10 +194,14 @@ def load_via_dbt_ls(self) -> None:
                     config=node_dict["config"],
                 )
                 nodes[node.unique_id] = node
+                logger.info("Parsed dbt resource `%s` of type `%s`", node.unique_id, node.resource_type)
```

Comment:
maybe this should be debug? sounds like this might be a bit too verbose?

Reply:
Done!